### ISSUE TITLE
Config-validator failed when inspecting security-config.yaml

### ISSUE THREAD

**Issue by dankirkp** [aws-samples/landing-zone-accelerator-on-aws-for-tse-se#6]
Running into an error when executing the AWSAccelerator-Pipeline.  The issue occurs in Build phase during the configuration validation.  Changes had been made to the region to target **us-east-1** as HOME_REGION, and references to **us-east-1** in the security_config.yaml were modified to remove them from all **excludeRegions** 

> [Container] 2024/09/15 15:35:52.696515 Running command yarn validate-config $CODEBUILD_SRC_DIR_Config
> yarn run v1.22.22
> $ ts-node ./packages/@aws-accelerator/accelerator/lib/config-validator.ts /codebuild/output/src2585/src/s3/01
> 2024-09-15 15:36:12.972 | info | config-validator | Found replacement variables in global-config.yaml
> 2024-09-15 15:36:12.972 | info | config-validator | Config source directory -  /codebuild/output/src2585/src/s3/01
> 2024-09-15 15:36:13.230 | info | replacements-config | Loading replacements config substitution values
> 2024-09-15 15:36:13.739 | info | accounts-config-validator | accounts-config.yaml file validation started
> 2024-09-15 15:36:13.740 | info | customizations-config-validator | customizations-config.yaml file validation started
> 2024-09-15 15:36:13.741 | info | global-config-validator | global-config.yaml file validation started
> 2024-09-15 15:36:13.743 | info | global-config-validator | email count: 1
> 2024-09-15 15:36:13.743 | info | global-config-validator | email count: 1
> 2024-09-15 15:36:13.743 | info | global-config-validator | email count: 1
> 2024-09-15 15:36:13.743 | info | global-config-validator | email count: 1
> 2024-09-15 15:36:13.744 | info | iam-config-validator | iam-config.yaml file validation started
> 2024-09-15 15:36:13.746 | info | network-config-validator | network-config.yaml file validation started
> 2024-09-15 15:36:13.984 | info | organization-config-validator | organization-config.yaml file validation started
> 2024-09-15 15:36:13.985 | info | security-config-validator | security-config.yaml file validation started
> 2024-09-15 15:36:13.991 | info | replacement-config-validator | replacements-config.yaml file validation started
> 2024-09-15 15:36:13.992 | warn | config-validator | Config file validation failed !!!
> 2024-09-15 15:36:14.007 | warn | config-validator | security-config.yaml has 1 issues:
> Duplicate AWS Config rules name exist with the same name and must be unique when deployed to the same account and region. Config rules in file: AWSAccelerator-s3-bucket-server-side-encryption-enabled,AWSAccelerator-s3-bucket-enforce-https,AWSAccelerator-attach-ec2-instance-profile,AWSAccelerator-ec2-instance-profile-permission,AWSAccelerator-elb-logging-enabled,AWSAccelerator-sagemaker-notebook-instance-kms-key-configured,AWSAccelerator-wafv2-logging-enabled,AWSAccelerator-dynamodb-in-backup-plan,AWSAccelerator-sagemaker-endpoint-configuration-kms-key-configured,AWSAccelerator-ebs-in-backup-plan,AWSAccelerator-rds-in-backup-plan,AWSAccelerator-elb-acm-certificate-required,AWSAccelerator-securityhub-enabled,AWSAccelerator-ec2-managedinstance-patch-compliance-status-check,AWSAccelerator-internet-gateway-authorized-vpc-only,AWSAccelerator-dynamodb-table-encrypted-kms,AWSAccelerator-acm-certificate-expiration-check,AWSAccelerator-alb-waf-enabled,AWSAccelerator-guardduty-non-archived-findings,AWSAccelerator-cloudtrail-enabled,AWSAccelerator-cloudtrail-security-trail-enabled,AWSAccelerator-cloudwatch-alarm-action-check,AWSAccelerator-cw-loggroup-retention-period-check,AWSAccelerator-db-instance-backup-enabled,AWSAccelerator-ec2-instance-detailed-monitoring-enabled,AWSAccelerator-elb-cross-zone-load-balancing-enabled,AWSAccelerator-incoming-ssh-disabled,AWSAccelerator-ec2-instances-in-vpc,AWSAccelerator-restricted-incoming-traffic,AWSAccelerator-s3-bucket-versioning-enabled,AWSAccelerator-vpc-sg-open-only-to-authorized-ports,AWSAccelerator-redshift-cluster-configuration-check,AWSAccelerator-s3-bucket-policy-grantee-check,AWSAccelerator-iam-user-group-membership-check,AWSAccelerator-cloudtrail-s3-dataevents-enabled,AWSAccelerator-iam-group-has-users-check,AWSAccelerator-api-gw-cache-enabled-and-encrypted,AWSAccelerator-ec2-volume-inuse-check,AWSAccelerator-elasticache-redis-cluster-automatic-backup-check,AWSAccelerator-emr-kerberos-enabled
> error Command failed with exit code 1.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> [Container] 2024/09/15 15:36:14.128225 Command did not exit successfully yarn validate-config $CODEBUILD_SRC_DIR_Config exit status 1
> [Container] 2024/09/15 15:36:14.131825 Phase complete: BUILD State: FAILED

**Comment by ghost** [comment]
@dankirkp I am running into this exact issue. what was the resolution ? I do not see any comments

**Comment by dankirkp** [comment]
It was an input error.  The yaml files are case-sensitive, and one character (I think it my accounts the I modified) were incorrect.  To I didn't realize what the issue was until I started with a fresh download of the TSE yaml files, and made my edits, redeployed, and it worked.  My advice would be to do the same - keep your yaml files that you have already edited, download a fresh copy of the configs, and make your changes again to the fresh copy.  Once done do a comparison and my guess is that you will find your issue.
